client/src/components/SelectInstrument.jsx

```diff
@@ -39,9 +39,10 @@ const SelectInstrument = ({ handleSelect, handleClick, size, ownInstrument, extr
         wrapAround
       >
         {
-          instruments.concat(extraInstruments.map(a => {
-            return `Your Instrument: ${a.instrumentName}`;
-          })).map(instrument => (
+          instruments.concat(extraInstruments.map(a => (
+             `Your Instrument: ${a.instrumentName}`
+          ))).map(instrument => (
+          instrument[0]==="Y"?<h1>{instrument}</h1>:
             <img
               src={`/assets/${instrument}.svg`}
```

Comment:
No images. just the name with be displayed as an h1. I can show an empty piano...